gym/lib/gym/options.rb

```diff
@@ -195,7 +195,51 @@ def self.plain_options
                                      env_name: "SUPPRESS_OUTPUT",
                                      description: "Suppress the output of xcodebuild to stdout. Output is still saved in buildlog_path",
                                      optional: true,
-                                     is_string: false)
+                                     is_string: false),
+        FastlaneCore::ConfigItem.new(key: :disable_xcpretty,
+                                     short_option: "-h",
+                                     env_name: "DISABLE_XCPRETTY",
+                                     description: "Disable xcpretty formatting of build output",
+                                     optional: true,
+                                     is_string: false),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_test_format,
+                                     short_option: "-t",
+                                     env_name: "XCPRETTY_TEST_FORMAT",
+                                     description: "Use the test (RSpec style) format for build output",
+                                     optional: true,
+                                     is_string: false),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_formatter,
+                                     short_option: "-v",
+                                     env_name: "XCPRETTY_FORMATTER",
+                                     description: "A custom xcpretty formatter to use",
+                                     optional: true,
+                                     verify_block: proc do |value|
+                                       UI.user_error!("Formatter file not found at path '#{File.expand_path(value)}'") unless File.exist?(value)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_report_junit,
+                                     short_option: "-1",
```

Comment:
Hey @bgannin! I think we should just remove the `short_option`s all together. They are not required by fastlane, and we should really try to keep them only for _very_ commonly used settings so as to not bring about collisions.
🚀